windows/ReactTestApp/Manifest.cpp

```diff
@@ -0,0 +1,42 @@
+#include "pch.h"
+
+#include "Manifest.h"
+
+#include <fstream>
+#include <iostream>
+
+namespace winrt::ReactTestApp::implementation
+{
+    template <typename T>
+    std::optional<T> try_get(const nlohmann::json &j, const std::string &key)
+    {
+        try {
+            return j.at(key).get<T>();
+        } catch (const nlohmann::json::exception &) {
+            return std::nullopt;
+        }
+    }  // namespace ReactTestApp
```

Comment:
This comment no longer applies
```suggestion
    }
```